### PR TITLE
Put ViewCase into a modal

### PR DESCRIPTION
**Pull request by allysonjp715**
Pulls the modal logic out into a separate component to be shared.

### CODE REVIEW

**Comment by allysonjp715** [comment]
[Bug fixes](https://trello.com/c/DKJztzhL/276-bug-fixes)

**Comment by allysonjp715** [comment]
Addresses some of #475

File: verification/curator-service/ui/src/components/AppModal.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+import CloseIcon from '@material-ui/icons/Close';
+import { WithStyles } from '@material-ui/core/styles/withStyles';
+import { withStyles } from '@material-ui/core';
+
+const styles = (theme: Theme) =>
+    createStyles({
+        modalContents: {
+            backgroundColor: theme.palette.background.paper,
+            left: '300px',
```
Comment (axmb) [inline verification/curator-service/ui/src/components/AppModal.tsx:19]: Is this relatively safe cross-platform/resolution?

Comment (allysonjp715) [reply verification/curator-service/ui/src/components/AppModal.tsx:19]: Should be fine on all desktop/laptop. Mobile is a good question but I don't think we're targeting mobile yet. I assume we'll need a bit of a bugbash if we want to make everything work on mobile.

**Pull request merged**